dao/src/main/java/com/fitgoal/dao/mybatis/config/MyBatisConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.fitgoal.dao.mybatis.config;
+
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+
+import java.io.IOException;
+import java.io.Reader;
+
+public class MyBatisConfig {
+
+    private static SqlSessionFactory sessionFactory;
+
+    private MyBatisConfig() {
+
+    }
+
+    static {
+        try (Reader reader = Resources.getResourceAsReader("mybatis/mybatis-config.xml")) {
+            sessionFactory = new SqlSessionFactoryBuilder().build(reader);
```

Comment:
This should be initialized in the application level

Reply:
Replaced